src/core/zowe/core_for_zowe_sdk/session.py

```diff
@@ -64,9 +66,14 @@ def __init__(self, props: dict) -> None:
         elif props.get("tokenValue") is not None:
             self.session.tokenValue = props.get("tokenValue")
             self.session.type = session_constants.AUTH_TYPE_BEARER
+        elif props.get("cert") is not None and props.get("certKey") is not None:
+            self.session.cert = props.get("cert")
+            self.session.certKey = props.get("certKey")
```

Comment:
I believe the properties we would want to use are `certFile` and `certKeyFile`, but only here.
I think it's fine to keep it as `cert` and `certKey` in the `ISession` object 😋